src/ui/src/components/core/content/CoreAnnotatedText.vue

```diff
@@ -41,8 +51,10 @@ import {
 	buttonTextColor,
 	cssClasses,
 	primaryTextColor,
+	createEnableCopyButtonField,
 } from "@/renderer/sharedStyleFields";
 import SharedControlBar from "@/components/shared/SharedControlBar.vue";
+import WdsCopyClipboardButton from "@/wds/WdsCopyClipboardButton.vue";
```

Comment:
Since this component imports floating UI, it's better to make it lazy to split the bundle.

```suggestion
import { defineAsyncComponentWithLoader } from "@/utils/defineAsyncComponentWithLoader";

const WdsCopyClipboardButton = defineAsyncComponentWithLoader({
	loader: () => import("@/wds/WdsCopyClipboardButton.vue"),
});
```